microbench/src/main/java/io/netty/microbench/handler/ssl/AbstractSslEngineThroughputBenchmark.java

```diff
@@ -32,17 +34,19 @@
 
     protected ByteBuffer wrapSrcBuffer;
     private ByteBuffer wrapDstBuffer;
+    private ByteBufAllocator allocator;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Perhaps 'allocator' could be replaced by a local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ASingularField)